.changeset/neat-rings-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': minor
```

Comment:
```suggestion
'@backstage/create-app': patch
```

Since we are still in 0.x releases, `patch` means new features and `minor` means breaking changes.